src/mushroom-strategy.js

```diff
@@ -188,6 +188,7 @@ class MushroomStrategy {
       }
     }
 
+    // TODO: Check if default is hidden
```

Comment:
Hey @DigiLive,
The miscellaneous domain doesn't check if the default domain is hidden. I was thinking we can just add an if statement to check if it's hidden but we can also move it up line 177 and use the domain variable to check if it's hidden. What's your call?

Reply:
This is a separate bug and you should create another issue or PR for it.
Try to keep each issue or "topic of change" separate from each other.

E.g.: 1 to add the Lock-Card, 1 to fix the MediaPlayer-Card, 1 to fix Miscellaneous.